Documentation/Input/Controllers.md

```diff
@@ -0,0 +1,5 @@
+# Controllers
+
+Controllers are created and destroyed automatically by [**input providers**](InputProviders.md). Each controller type has a number of *physical inputs* defined by an *axis type*, telling us the data type of the input value (Digital, Single Axis, Dual Axis, Six Dof, ...), and an *input type* (Button Press, Trigger, Thumb Stick, Spatial Pointer, ...) describing the origin of the input. Physical inputs are mapped to *input actions* via in the **Controller Input Mapping Profile**, under the *Input System Profile* in the Mixed Reality Toolkit component.
```

Comment:
I see what you mean. I had been editing the files in Visual Studio Code that wraps thing around nicely so I didn't realize how horrible they are when seen here. I see that it is possible to break the lines while still having them display as a single paragraph when renderered. I don't know if it is worth the additional effort though.

Reply:
No worries, I had a change of heart here.

I think that our tooling isn't at the stage where asking people to break lines is reasonable, and we should improve our tooling so that this should be automatic for us.